.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: java
 jdk:
-  - oraclejdk8
+- oraclejdk8
 addons:
   sonarcloud:
-    organization: "improbable-keanu"
-script:
-  - ./gradlew check
-  - 'if [[ ! -z $SONAR_TOKEN ]]; then sonar-scanner; fi'
+    organization: improbable-keanu
+script: deployment/buildAndDeploy.sh
+before_install:
+- openssl aes-256-cbc -K $encrypted_eec04b57ffc0_key -iv $encrypted_eec04b57ffc0_iv
+  -in deployment/secret-keys-keanu.gpg.enc -out deployment/secret-keys-keanu.gpg -d
```

Comment:
How does this work? Is `$encrypted_eec04b57ffc0_key` something that lives on the Travis server that nobody can steal?

Reply:
Yep exactly, when you run `travis encrypt` on a file it creates that `.enc` version of the file and creates these two environmental variables for you.

For more: https://docs.travis-ci.com/user/encrypting-files/

Reply:
👍